net.sourceforge.wxEDID.yml

```diff
@@ -0,0 +1,49 @@
+app-id: net.sourceforge.wxEDID
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+command: wxedid
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host:ro
```

Comment:
this is a very broad file system permission.

Reply:
It seems common for wxWidgets based flatpaks.
Would `--filesystem=home:ro` be better?

Reply:
"others do it" isn't a proper answer.

Reply:
Sorry, the reason is wxWidgets wxFileDialog doesn't seem support portals.

Reply:
I've created a patched for wxWidgets to use `GtkFileChooserDialog`.